lib/dialects/mssql/query-generator.js

```diff
@@ -725,6 +725,25 @@ const QueryGenerator = {
     });
   },
 
+  getCheckConstraintQuery(tableName, attributeName) {
+    const sql = 'SELECT name FROM SYS.CHECK_CONSTRAINTS ' +
```

Comment:
Use ES6 template string substitution instead of `lodash _.template`. So where appropriate 

```js
`SOME SQL STATEMENT ${tableName}` +
`SOME SQL STATEMENT ${columnName}`
```

Reply:
On another thought, you should be able to get this constraint using `showConstraint` cc @harshithkashyap